src/get_dem.py

```diff
@@ -105,18 +115,19 @@ def get_tile_for(args):
     with open(myfile) as f:
         content = f.readlines()
         for item in content:
-	    if demname in item.split()[0] and len(demname) == len(item.split()[0]):
-	        (mydir,myfile) = os.path.split(item)
-	        mydir = mydir.split()[1]
-	        if "s3" in mydir:
-		    myfile = os.path.join(demname,fi)+".tif"
+            if demname in item.split()[0] and len(demname) == len(item.split()[0]):
+                (mydir,myfile) = os.path.split(item)
+                mydir = mydir.split()[1]
+                if "s3" in mydir:
+                    myfile = os.path.join(demname,fi)+".tif"
                     s3 = boto3.resource('s3')
+                    resource.meta.client.meta.events.register('choose-signer.s3.*', disable_signing)
```

Comment:
Crashes with `NameError: name 'resource' is not defined`, should be `s3.meta.client`